pycontw/pycontw-frontend
# feat(header/locale): add icon and adjust

## Types of changes


* **New feature**


## Description

Add globe icon and adjust language name for language switch on header.

## Steps to Test This Pull Request

1. Go to "http://localhost:3000/2024/zh-hant"
2. Check if new icon is added
3. Check if language name is consistent

## Related Issue

#520 

## Screenshot

<img width="947" alt="image" src="https://github.com/pycontw/pycontw-frontend/assets/26858727/a73686cb-3a1f-4fd3-976f-5beddef492f8">
<img width="119" alt="image" src="https://github.com/pycontw/pycontw-frontend/assets/26858727/d4c78642-6bc0-4f38-a2ae-b8c00e8e4190">
